src/components/Richtext/RichTextContainer.tsx

```diff
@@ -50,14 +52,18 @@ export const RichTextContainer: React.FC = () => {
     setOpenPromptDrawer(false);
   };
 
+  useEffect(() => {
```

Comment:
bisa kasih todo note ga disini?
kayaknya masih bisa di improve, tapi worked for now